<?xml version="1.0"?>

<launch>
    <!-- local params -->
    <rosparam file="$(find traj_pred)/config/traj_pred_node.yaml" />

    <!-- launch trajectory predictor -->
    <node pkg= "traj_pred" name="traj_pred_node" type="traj_pred_node.py"  output="screen">
    </node>
</launch>
